expyfun/visual/_visual.py

```diff
@@ -991,25 +991,10 @@ def __init__(self, ec, file_name, pos=(0, 0), units='norm', scale=1.,
         self._units = units
         self._center = center
         self.set_scale(scale)  # also calls set_pos
-        self.set_visible(visible)
-        self._update_background()
-
-    def _update_background(self):
-        from pyglet.image import ImageData
-        from pyglet.gl import GLubyte
-        _px = np.tile(self._ec._bgcolor,
-                      self._ec.window_size_pix.prod()).astype(int).tolist()
-        self._background = ImageData(*self._ec.window_size_pix, format='RGBA',
-                                     data=(GLubyte * len(_px))(*_px))
-
-    def play(self, autoshow=True):
-        """Play video from current position.
+        self._visible = visible
 
-        Parameters
-        ----------
-        autoshow : bool
-            If ``True`` (and video is currently hidden), will make video
-            visible before playing.
+    def play(self):
```

Comment:
`play(autoshow)` and `pause(autohide)` were causing more problems than they were worth.